packages/backend-storage/src/storage_access_orchestrator.ts

```diff
@@ -1,41 +1,34 @@
 import {
   ConstructFactoryGetInstanceProps,
-  SsmEnvironmentEntriesGenerator,
+  SsmEnvironmentEntry,
 } from '@aws-amplify/plugin-types';
 import { StorageAccessDefinition, StoragePath } from './types.js';
-import { IBucket } from 'aws-cdk-lib/aws-s3';
-import { StorageAccessPolicyFactory } from './storage_access_policy_factory.js';
+import { StorageAccessArbiter } from './storage_access_arbiter.js';
 import { ownerPathPartToken } from './constants.js';
-import { AcceptorTokenAccessMap } from './action_to_resources_map.js';
 
 /**
  * Middleman between creating bucket policies and attaching those policies to corresponding roles
  */
-export class StorageAccessPolicyArbiter {
+export class StorageAccessOrchestrator {
```

Comment:
Also open to name suggestions here

Reply:
`StorageAccessIAMPolicyCreator/Generator` ?

Reply:
With my latest refactor I actually think orchestrator is a pretty good name for this class. But still open to other suggestions